src/deploy/osp_deployer/settings/gen_scripts/gen_ini_file.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
+
+# Copyright (c) 2018 Dell Inc. or its subsidiaries.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################################################################
+# Run this script run from the director node as the director's admin user.
```

Comment:
Duplicate "run".  "Run this script from the director ..."?

Reply:
Actually, that was copied in with the copyright header and needs to go away.